src/electron-forge-start.js

```diff
@@ -10,7 +10,9 @@ import { start } from './api';
   program
     .version(require('../package.json').version)
     .arguments('[cwd]')
+    .option('-p, --app-path <path>', 'Override the path to the electron app to launch (defaults to \'.\')')
```

Comment:
Does our linter setup complain if the description string here is double-quoted? I want to avoid escaped quotes for readability.